columnphysics/icepack_parameters.F90

```diff
@@ -1609,6 +1633,8 @@ subroutine icepack_recompute_constants()
 
 !autodocument_end
 
+      real (kind=dbl_kind) :: lambda
```

Comment:
Is lambda just a temporary variable, or does it have a physical meaning?  units?

Reply:
It is unit-less. I followed our in-house implementation, which in turn follows Roy et al 2015. Lambda is defined in equation (6) (top of p. 7).